pyfr/plugins/base.py

```diff
@@ -91,3 +94,111 @@ def _invoke_postaction(self, **kwargs):
 
     def __call__(self, intg):
         pass
+
+
+class RegionMixin(object):
+    def _init_writer_for_region(self, intg, nout, prefix):
+        # Base output directory and file name
+        basedir = self.cfg.getpath(self.cfgsect, 'basedir', '.', abs=True)
+        basename = self.cfg.get(self.cfgsect, 'basename')
+
+        # Region of interest
+        region = self.cfg.get(self.cfgsect, 'region', '*')
+
+        # All elements
+        if region == '*':
+            mdata = self._prepare_mdata_all(intg, nout, prefix)
+            self._add_region_data = self._add_region_data_all
+        # All elements inside a box
+        elif ',' in region:
```

Comment:
Boundary names are allowed to have ',' in them so in some rare cases it could interfere with the box region tag. Possibly elif all(x in region for x in ['(', ',', ')']) ?

Reply:
I think this should be resolved now, thanks.